apps/server/src/modules/fwu/fwu.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { AuthorizationModule } from '@src/modules/authorization';
+import { Logger } from '@src/core/logger';
+import { FwuController } from './controller/fwu.controller';
+import { FwuUc } from './uc/fwu.uc';
+
+@Module({
+	imports: [AuthorizationModule],
+	controllers: [FwuController],
+	providers: [FwuUc, Logger],
+	exports: [FwuUc],
```

Comment:
we should not export the UC
instead, this can be split into uc and service, and only the service can be exported
however, why you need to export it in the first place? Which other module needs this?

Reply:
Deleted the exports in line 11.